<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server</class>
 <widget class="QMainWindow" name="Server">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>554</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>228</width>
    <height>554</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>228</width>
    <height>554</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Server</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>191</width>
      <height>481</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>5</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>9</pointsize>
         <italic>false</italic>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
       </property>
       <property name="title">
        <string>CREATE SERVER</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <widget class="QWidget" name="layoutWidget_3">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>45</y>
          <width>181</width>
          <height>291</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="font">
            <font>
             <family>MS Shell Dlg 2</family>
             <pointsize>9</pointsize>
             <italic>false</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="whatsThis">
            <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;PORT:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>PORT:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="portLine">
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 0, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maxPlayersLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>MAX PLAYERS:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="maxPlayersLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="maxLoginLengthLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>MAX LOGIN LENGTH:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="MaxLoginLengthLine">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="minLoginLengthLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
           </property>
           <property name="text">
            <string>MIN LOGIN LENGTH:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="minLoginLength">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CreateBtn">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>CREATE</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="StopBtn">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>STOP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RestartBtn">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>RESTART</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="DeleteAllClientBtn">
       <property name="styleSheet">
        <string notr="true">color:rgb(0, 05, 0);
font: 75 9pt &quot;MS Shell Dlg 2&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>DELETE ALL</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>228</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
